nixos/modules/services/misc/klipper.nix

```diff
@@ -156,7 +222,7 @@ in
           (mcu: firmware: pkgs.klipper-flash.override {
             mcu = lib.strings.sanitizeDerivationName mcu;
             klipper-firmware = firmware;
-            flashDevice = cfg.settings."${mcu}".serial;
+            flashDevice = default cfg.firmwares."${mcu}".serial cfg.settings."${mcu}".serial;
```

Comment:
Why is this change needed?

Reply:
Because otherwise there wouldn't be a way to specify serial port if you use `configFile` instead of `settings `